frontend/widgets/OBSBasic.hpp

```diff
@@ -447,6 +447,7 @@ public slots:
 	void RemoveDockWidget(const QString &name);
 	bool IsDockObjectNameUsed(const QString &name);
 	void AddCustomDockWidget(QDockWidget *dock);
+	void setDockCornersVertical(bool vertical);
```

Comment:
```suggestion
	void SetDockCornersVertical(bool vertical);
```
New functions should match our casing, it also allows to distinguish from library functions (e.g. Qt, C++ STL).

Reply:
I JUST mentioned this to someone else on their PR earlier today god damnit

Reply:
Wait no, this casing is correct per @PatTheMav's guidelines in https://github.com/obsproject/obs-studio/pull/11622